headers.go

```diff
@@ -4,10 +4,10 @@ import (
 	"net/http"
 )
 
-type Header struct {
+type Headers struct {
```

Comment:
type Headers struct {
    map[string]string
    req *http.Request
}

Try it, if can't, just give up.